src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -563,9 +580,42 @@ void RenderTarget::applyCurrentView()
 {
     // Set the viewport
     const IntRect viewport = getViewport(m_view);
-    const int     top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
+    int           top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
     glCheck(glViewport(viewport.left, top, viewport.width, viewport.height));
 
+    const FloatRect scissor = m_view.getScissor();
+
+    if (m_cache.scissorEnabled)
```

Comment:
Is this differentiation even necessary? Feels like unnecessary code duplication just to avoid one potentially unnecessary call to `glDisable()` by doing one extra branching. And if we go that far, shouldn't it verify the rectangle actually changed since it's been enabled?

Reply:
Fixed.